.vscode/settings.json

```diff
@@ -1,13 +1,14 @@
 {
-	"[typescript]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
-		"editor.formatOnSave": true
-	},
-	"[typescriptreact]": {
-		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
-		"editor.formatOnSave": true
-	},
-	"eslint.run": "onType",
-	"eslint.format.enable": true,
-	"typescript.tsdk": "node_modules/typescript/lib"
-}
```

Comment:
.vscode should be .gitignored and deleted to avoid these conflicts

Reply:
Actually, your `.vscode/settings.json` and my `.vscode/settings.json` are the same, just that mine has the setting `files.eol` set to `\n`. I don't really think the whole folder should be git ignored...

Reply:
In that case you could mark the files as unchanged locally and undo the changes on this branch, these settings should serve as a base on which to apply personal settings on.